src/js/TuningSliders.js

```diff
@@ -207,7 +210,6 @@ TuningSliders.resetDTermFilterSlider = function() {
     }
 
     this.calculateNewDTermFilters();
-    this.updateFilterSlidersDisplay();
```

Comment:
Maybe this one should stay

Reply:
With above changes we don't have to press the enable the DTerm slider twice after disabling Dterm settings in legacy mode.